tests/unit/batch_sys_handlers/test_slurm.py

```diff
@@ -32,10 +35,12 @@
                 'task_id': 'axe.1',
             },
             [
-                '#SBATCH --job-name=axe.1.chop',
-                '#SBATCH --output=cylc-run/chop/log/job/1/axe/01/job.out',
-                '#SBATCH --error=cylc-run/chop/log/job/1/axe/01/job.err',
-                '#SBATCH --time=3:00',
+                f'#SBATCH --job-name=axe.1.chop',
```

Comment:
Minor point: Reviewers may want to consider the Python style here:
* Is it ok to mix follow-on lines with new lines in the list?
* Is it ok to make all the strings f-strings - I think it looks better because they all have the same indent, but some style guides say not to do it if you don't have any substitutions in them.

Reply:
Hmmm my personal preference is not to use the f string unless there is a substitution - my brain automatically looks for the substitution when I see the f. Not a deal breaker though if you prefer them.

Reply:
I've had a double check with black(happy) and flake8(unhappy) and failed to get pylint to recognize the f-strings are valid syntax. I'm going to leave these for now.